tributary/tests/streaming/output/test_kafka_streaming.py

```diff
@@ -1,7 +1,9 @@
 import tributary.streaming as ts
+import pytest
 
 
 class TestKafka:
+    @pytest.mark.skipif("sys.platform != 'linux'")
```

Comment:
these tests work on any platform, we just dont have azure setup for them correct? probably better to set an env var on azure and skip if its set, like `TRIBUTARY_SKIP_DOCKER_TESTS`